angular/lib/src/lib/slider/slider.component.ts

```diff
@@ -0,0 +1,539 @@
+import type {SliderHandleState, SliderState} from '@agnos-ui/angular-headless';
+import {callWidgetFactory, createSlider, patchSimpleChanges, toSlotContextWidget, UseDirective} from '@agnos-ui/angular-headless';
+import type {SliderActions} from '@agnos-ui/core';
+import {NgFor, NgIf} from '@angular/common';
+import type {AfterViewInit, OnChanges, OnDestroy, Signal, SimpleChanges} from '@angular/core';
+import {
+	ChangeDetectionStrategy,
+	Component,
+	DestroyRef,
+	ElementRef,
+	EventEmitter,
+	forwardRef,
+	inject,
+	Input,
+	NgZone,
+	Output,
+	ViewChild,
+	ViewEncapsulation,
+} from '@angular/core';
+import {toSignal} from '@angular/core/rxjs-interop';
+import {NG_VALUE_ACCESSOR} from '@angular/forms';
+import {take} from 'rxjs';
+
+/**
+ * A class that is responsible for:
+ * the click and keyboard even capture and propagation
+ * positioning of the slider handle
+ */
+@Component({
+	selector: '[auSliderHandle]',
+	exportAs: 'auSliderHandle',
+	standalone: true,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	host: {
+		class: 'au-slider-handle',
+		'[class]': "state().vertical ? 'au-slider-handle-vertical' : 'au-slider-handle-horizontal'",
+		'[style.left.%]': 'state().positionLeft[state().handleId]',
+		'[style.top.%]': 'state().positionTop[state().handleId]',
+		'(keydown)': 'widget.actions.onKeyDown($event, state().handleId)',
+		'(mousedown)': 'widget.actions.onMouseDown($event)',
+	},
+	hostDirectives: [
```

Comment:
`hostDirectives: [UseDirective]`

Reply:
oh, i thought I removed this directive, no need for it actually

Reply:
even better :)